spec/jekyll-remote-theme/mock_gemspec_spec.rb

```diff
@@ -1,15 +1,63 @@
 # frozen_string_literal: true
 
 RSpec.describe Jekyll::RemoteTheme::MockGemspec do
+  let(:fixture) { "alldeps" }
+  let(:contents) { File.read gemspec_dir("#{fixture}.gemspec") }
+  let(:filename) { "#{theme.name}.gemspec" }
+  let(:path) { File.expand_path filename, theme.root }
   let(:nwo) { "pages-themes/primer" }
   let(:theme) { Jekyll::RemoteTheme::Theme.new(nwo) }
   subject { described_class.new(theme) }
 
-  it "returns an empty array of runtime dependencies" do
-    expect(subject.runtime_dependencies).to eql([])
+  before { File.write path, contents }
+
+  it "stores the theme" do
+    expect(subject.send(:theme)).to eql(theme)
+  end
+
+  it "determines the path" do
+    expect(subject.send(:path)).to eql(path)
+  end
+
+  it "reads the contents" do
+    expect(subject.send(:contents)).to eql(contents)
+  end
+
+  it "builds potential_paths" do
+    expect(subject.send(:potential_paths)).to include(path)
   end
 
   it "returns the theme root" do
     expect(subject.full_gem_path).to eql(theme.root)
   end
+
+  context "fixtures" do
+    let(:dependency_names) { subject.send(:dependency_names) }
+    let(:runtime_dependencies) { subject.runtime_dependencies }
+
+    # Hash in the form of gemspec fixture => expected dependencies
+    {
+      "alldeps" => %w(jekyll jekyll-feed jekyll-sitemap bundler rake),
```

Comment:
`bundler` and `rake` are usually development dependencies. We don't want to require those, right?

Reply:
In the fixture, they're listed as runtime.